docs/reference/api/usb/usb_component.md

```diff
@@ -0,0 +1,89 @@
+<h2 id="usb-overview">USB components overview</h2>
+
+The Universal Serial Bus (USB) is the most widely used bus in today's computer. USB has been designed to standardize connections between the computer and peripherals. For example, keyboards, mice, USB audio devices, printers, scanners, disk drives and cameras can use the same bus to exchange data with a computer.
+
+This document refers to the Mbed OS classes providing USB peripheral functionality as USB components. These USB components inherit from USBDevice and provide specific USB peripherial functionality. Below is information common to all of the Mbed OS USB components and how to use them.
+
+### Component construction
+
+Constructing a USB component with the default parameters perform initialization for you. Once the constructor is finished then the device is ready to be used. This process:
```

Comment:
Query: What does this first sentence mean? Please clarify.
Query: In the second sentence, who or what finishes the constructor? The device is ready to be used by whom or by what? You, the user? Please clarify.